### PR TITLE
Feature/ml 499 activations benchmark

### PR DESCRIPTION
**Pull request by iryna-melnyk**
some test changes

### CODE REVIEW

File: libs/vm-modules/benchmark/model/model.cpp [hunk 1]
```diff
@@ -77,812 +77,896 @@ fetch::vm::Ptr<fetch::vm_modules::ml::model::VMModel> vmSequentialModel(VMPtr &v
...
-  auto model = vmSequentialModel(vm, config.sizes, config.activations, "mse", "adam");
-
-  // serialise to string
-  fetch::vm::Ptr<fetch::vm::String> serialized_model = model->SerializeToString();
-
-  auto new_model = vmSequentialModel(vm);
-  state.counters["charge"] =
-      static_cast<double>(new_model->Estimator().DeserializeFromString(serialized_model));
-  state.counters["StrLen"] = static_cast<double>(serialized_model->string().size());
+  // Get config
```
Comment (iryna-melnyk) [inline libs/vm-modules/benchmark/model/model.cpp:739]: look here. don't mind the commented code

**Review by vhavrylov-ncube — COMMENTED** [review]
Looks good to me, let's wait for others.

File: libs/vm-modules/benchmark/model/model.cpp [hunk 2]
```diff
@@ -694,6 +694,98 @@ BENCHMARK(BM_Fit)
...
+    auto res = model->Predict(data);
+  }
+}
+
+BENCHMARK(BM_AddActivation)->Args({1, 1, getId("relu")})->Unit(::benchmark::kMicrosecond);
+BENCHMARK(BM_AddActivation)->Args({10, 1, getId("relu")})->Unit(::benchmark::kMicrosecond);
+BENCHMARK(BM_AddActivation)->Args({100, 1, getId("relu")})->Unit(::benchmark::kMicrosecond);
+BENCHMARK(BM_AddActivation)->Args({1000, 1, getId("relu")})->Unit(::benchmark::kMicrosecond);
+BENCHMARK(BM_AddActivation)->Args({10, 10, getId("relu")})->Unit(::benchmark::kMicrosecond);
+BENCHMARK(BM_AddActivation)->Args({100, 10, getId("relu")})->Unit(::benchmark::kMicrosecond);
```
Comment (MissingNO57) [inline libs/vm-modules/benchmark/model/model.cpp]: It's good, please just add benchmarks for other activations.

**Review by MissingNO57 — CHANGES_REQUESTED** [review]

File: libs/vm-modules/benchmark/model/model.cpp (hunk 2 above)
Comment (iryna-melnyk) [reply libs/vm-modules/benchmark/model/model.cpp]: done

**Review by baykaner — APPROVED** [review]

**Review by baykaner — APPROVED** [review]

**Review by MissingNO57 — APPROVED** [review]
LGTM

**Pull request merged**